dev/tools/controllerbuilder/config/v1beta1/managedkafka.yaml

```diff
@@ -18,3 +18,5 @@ generateMapper: true
 resources:
   - kind: ManagedKafkaCluster
     protoName: Cluster
+  - kind: ManagedKafkaTopic
```

Comment:
We should deprecate the config files since we're taking a different approach to validating generated files in presubmit.

Reply:
I like that we show different approaches for the same problem (some resources could fit one or the either). Is this a blocker to introduce frictions if not deprecating?